src/github.com/mozilla-services/pushgo/simplepush/error.go

```diff
@@ -63,8 +66,8 @@ func ErrToStatus(err error) (status int, message string) {
 	if err == nil {
 		return http.StatusOK, ""
 	}
-	if code, ok := err.(ErrorCode); ok {
-		switch status = code.Status(); status {
+	if se, ok := err.(*ServiceError); ok {
+		switch status = se.Status(); status {
```

Comment:
There's slight risk in exposing different levels of authority when returning errors to malicious parties, but I don't think that it's fully warranted here. Exposing that status exists, but the user is not authorized (or other errors) should be fine.